tedana/tests/test_model_monoexponential.py

```diff
@@ -0,0 +1,43 @@
+"""
+Tests for tedana.model.monoexponential
+"""
+import os.path as op
+
+import numpy as np
+
+from tedana import utils
+from tedana.model import monoexponential as me
+from tedana.tests.utils import get_test_data_path
+
+
+def test_fit_decay():
+    tes = np.array([14.5, 38.5, 62.5])
+    in_files = [op.join(get_test_data_path(), 'echo{0}.nii.gz'.format(i+1))
+                for i in range(3)]
+    data, _ = utils.load_data(in_files, n_echos=len(tes))
+    mask, mask_sum = utils.make_adaptive_mask(data, minimum=False, getsum=True)
```

Comment:
Is there a way to refactor this to avoid duplicating the same 5 lines (`tes = ...` through `mask, mask_sum = ...`)? Options include making them globals or creating a class structure with those lines in the `__init__` and having `test_fit_decay` and `test_fit_decay_ts` as methods.

Indeed, this goes back to #50... Having to repeat these steps for testing everything seems wasteful / redundant.

Reply:
Oof. This might be outside the scope of this PR, since #50 is such an extensive issue. 

Personally, I'd rather repetition than re-introducing globals at this point&mdash; I know that their usage is less problematic here, but we just went through so much trouble to get rid of them ! It's mostly my being dogmatic, but still.

@tsalo, if you can see a clean way to cut down on the repetition, that would be great. If you want to postpone tackling this until we modularize the functions (much) better, I'm fine with that, too.

Reply:
Good catch @rmarkello. I'll just add a test-specific pytest fixture to load data.

Reply:
That's a great solution!!!